core/build.gradle

```diff
@@ -65,10 +100,15 @@ shadowJar {
     relocate 'org.apache.http', "${dependenciesPrefix}.http"
     relocate 'eu.infomas.annotation', "${dependenciesPrefix}.annotation"
     relocate 'org.apache.commons', "${dependenciesPrefix}.apachecommons"
-    relocate 'com.google.common', "${dependenciesPrefix}.googlecommon"
+//    relocate 'com.google.common', "${dependenciesPrefix}.googlecommon"
     relocate 'javax.annotation', "${dependenciesPrefix}.javaxannotation"
-    relocate 'com.google.gson', "${dependenciesPrefix}.gson"
-    relocate 'com.google.thirdparty', "${dependenciesPrefix}.google.thirdparty"
+//    relocate 'com.google.gson', "${dependenciesPrefix}.gson"
+//    relocate 'com.google.thirdparty', "${dependenciesPrefix}.google.thirdparty"
+    relocate 'com.google', "${dependenciesPrefix}.google"
+    relocate 'io.grpc', "${dependenciesPrefix}.io.grpc"
+    relocate 'io.opencensus', "${dependenciesPrefix}.io.opencensus"
```

Comment:
Indeed gRPC core has OpenCensus embedded in it. http://central.maven.org/maven2/io/grpc/grpc-core/1.14.0/grpc-core-1.14.0.pom (strange but seems like a force :))